src/collectors/mesos/mesos.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+
+"""
+Collect the Mesos stats for the local node.
+
+#### Dependencies
+ * urlib2
+"""
+
+import urllib2
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+import diamond.collector
+
+from diamond.collector import str_to_bool
+
+
+class MesosCollector(diamond.collector.Collector):
+    def __init__(self, config=None, handlers=[], name=None, configfile=None):
+        self.master = True
+        self.known_frameworks = {}
+        super(MesosCollector, self).__init__(config, handlers, name, configfile)
+
+    def process_config(self):
+        super(MesosCollector, self).process_config()
+        self.master = str_to_bool(self.config['master'])
+
+    def get_default_config_help(self):
+        config_help = super(MesosCollector,
+                            self).get_default_config_help()
+        config_help.update({
+            'host': "",
```

Comment:
Can you describe what those variables are? maybe something like "host running mesos daemon", "port on which the messos deamon is listening"